root/static/scripts/edit/URLCleanup.js

```diff
@@ -1290,7 +1295,7 @@ const CLEANUPS = {
       return url.replace(/^(?:https?:\/\/)?(?:[^\/]+\.)?lyric\.evesta\.jp\/([al]\w+\.html).*$/, 'http://lyric.evesta.jp/$1');
     },
     validate: function (url, id) {
-      const m = /^http:\/\/lyric\.evesta\.jp\/(a|l)\w+\.html$/.exec(url);
+      const m = /^http:\/\/lyric\.evesta\.jp\/[al]\w+\.html$/.exec(url);
       if (m) {
         const prefix = m[1];
```

Comment:
Won't this break, since `(a|l)` was `m[1]` here? (A test would be a good idea, if so.)

Reply:
Got it now, should be fixed.

Reply:
Adding adhoc test would be neat still.